### ISSUE TITLE
Dacryocytosis

### ISSUE THREAD

**Issue by jpgourdine** [obophenotype/human-phenotype-ontology#4859]
**Preferred term label:**
Dacryocytosis

**Synonyms**
Tear-drop shaped erythrocytes, dacryocytes

**Definition (free text, please give PubMed ID)**
Presence of teardrop-shaped red blood cells.
PMID: 21250106

**Parent term (use hpo.jax.org/app)**
Poikilocytosis (HP:0004447)
http://purl.obolibrary.org/obo/HP_0004447

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM : 254450 MYELOFIBROSIS

**Your nano-attribution (ORCID)**
0000-0001-9969-8610

**Comment by pnrobinson** [comment]
added as HP_0032563
